pkg/operator/controllers/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -32,50 +32,65 @@ var machinehealthcheckYaml []byte
 var mhcremediationalertYaml []byte
 
 const (
-	ControllerName string = "MachineHealthCheck"
-	managed        string = "aro.machinehealthcheck.managed"
-	enabled        string = "aro.machinehealthcheck.enabled"
+	MHCControllerName string = "MachineHealthCheck"
+	MHCManaged        string = "aro.machinehealthcheck.managed"
+	MHCEnabled        string = "aro.machinehealthcheck.enabled"
```

Comment:
nit: I think we can remove the `MHC` prefix changes here, since these constants will already be namespaced by the package name - we'd have `machinehealthcheck.MHCControllerName` which feels redundant compared to `machinehealthcheck.ControllerName`. I think this also breaks a standard pattern we have for all other controllers (seen in `operator.go`)